tao_compiler/mlir/disc/transforms/disc_lhlo_rewriter.cc

```diff
@@ -0,0 +1,156 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Not sure about the effect of Copy Right claim. Possiblely better to use the one of BladeDISC?